### PR TITLE
Update humans.txt

### LINKED ISSUE #6693: Update humans.txt
Please update https://wellcomecollection.org/humans.txt with this file (request from Hannah)

[humans.txt 2021.txt](https://github.com/wellcomecollection/wellcomecollection.org/files/6740228/humans.txt.2021.txt)

### PR DESCRIPTION
**Pull request by davidpmccormick**
☝️ 

Closes #6693

### CODE REVIEW

File: assets/humans.txt [hunk 1]
```diff
@@ -1,103 +1,107 @@
...
+	Senior User Interface Designer:	Gareth Ormerod
+	Product Designer:		Dominique Marshall
+	User Researcher:		Alice Richmond
+	Senior Software Developer:	Gareth Estchild
+	Software Developer:		David McCormick
+
+Digital Preservation and Data
+	Technical Product Manager:	Jonathan Tweed
+	Technical Delivery Manager:	Natalie Pollecutt
+	Name: 				James Gorrie
```
Comment (davidpmccormick) [inline assets/humans.txt:56]: @jamesgorrie I'm tidying up the spacing in this file and spotted this. Are you a 'Senior Software Developer'?

Comment (jamesgorrie) [reply assets/humans.txt]: I'd be happy as developer.

Comment (davidpmccormick) [reply assets/humans.txt]: your wish is my command

Comment (davidpmccormick) [reply assets/humans.txt]: …actually, latest update means nobody gets a job title which I guess (hope) you'll also be happy with.

**Pull request merged**